client/core/core.go

```diff
@@ -10755,3 +10755,59 @@ func (c *Core) checkEpochResolution(host string, mktID string) {
 
 	}
 }
+
+// RedeemGeocode redeems the provided game code with the wallet and redeems the
+// prepaid bond (code is a prepaid bond). If the user is not registered with
+// dex.decred.org yet, the dex will be added first.
+func (c *Core) RedeemGeocode(appPW, code []byte, msg string) (dex.Bytes, uint64, error) {
+	const dcrBipID = 42
+	dcrWallet, found := c.wallet(dcrBipID)
+	if !found {
+		return nil, 0, errors.New("no decred wallet")
+	}
+	if !dcrWallet.connected() {
+		return nil, 0, errors.New("decred wallet is not connected")
+	}
+
+	host := "dex.decred.org:7232"
+	switch c.net {
+	case dex.Testnet:
+		host = "bison.exchange:17232"
+	case dex.Simnet:
+		host = "127.0.0.1:17273"
+	}
+	cert := CertStore[c.net][host]
```

Comment:
Host should be a param. Maybe make it be part of the code?

Reply:
I agree that the server should be a parameter, but it can't be "part of the code". If we expand this, we'll just have to note the server on the paper in the cache. Leaving for now.

Reply:
Just to follow up, here is my reasoning for hard-coding the hosts.

- When we make the host a parameter, we will be adding more steps and new things to break.
- We could start with radio buttons for our "known servers", but there is only one public mainnet server right now anyway.
- We could add an input field for a custom host, but since we're so late in the release cycle, I want to minimize new code, and I think there's a balance to be struck